### ISSUE TITLE
No Automatic roll on dice

### ISSUE THREAD

**Issue by Saladin926** [thyer/CS340winter2016#660]
Dice should roll automatically after a little while

**Comment by cmriboldi** [comment]
This isn't a real bug. There is no where that it says that's what needs to happen. We decided that we didn't want that because we thought it was annoying.

**Comment by thyer** [comment]
No, this is valid, and it's from Phase 2. See the implementation notes at https://students.cs.byu.edu/~cs340ta/winter2016/group_project/Phase2ImplNotes.pdf which mention in number 8 that this is required.

**Comment by jvansteeter** [comment]
The specs say, "We have an automatic timer (which
the students will have to implement) for rolling, as well as the rolling button."
Where is this automatic timer in the client code located that I might go implement it?
